Author: Sebastian Harl <tokkee@debian.org>
Description: Bail out with an error if octave-signal is not available.

--- a/src/octave/pfsstat
+++ b/src/octave/pfsstat
@@ -33,6 +33,11 @@
   error( "Expecting no parameters" ); 
 endif
 
+if( 2 != exist( 'gausswin' ) )
+  error( "%s requires the 'signal' package - please install octave-signal\n", \
+      program_name () );
+endif
+
 firstFrame = true;
 while( true )
   pin = pfsget( pin );
